tests/searchcommands/test_configuration_settings.py

```diff
@@ -81,7 +81,7 @@ def generate(self):
 
         self.assertIs(command.configuration.distributed, False)
         self.assertIs(command.configuration.generating, True)
-        self.assertEqual(command.configuration.type, 'streaming')
+        self.assertEqual(command.configuration.type, 'stateful')
```

Comment:
I had to set this back to get the test to pass.
self.assertEqual(command.configuration.type, 'streaming')